packages/react-focus-management/src/FocusManagementProvider.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { makeMergeProps } from '@fluentui/react-utils';
+import { getCurrentAbilityHelpers, createAbilityHelpers, Types as AHTypes } from 'ability-helpers';
+import { internal__FocusManagementContext, FocusManagementContextValue } from './focusManagementContext';
+
+type Dir = 'ltr' | 'rtl';
+
+export interface FocusManagementProvideProps extends React.HTMLAttributes<HTMLElement> {
+  dir?: Dir;
+
+  window?: Window;
+
+  ahProps?: AHTypes.AbilityHelpersCoreProps;
```

Comment:
```suggestion
```

Do we need to expose this now?

Reply:
no, I think I will select the props I currently use and expose them instead like v0 does for popper